src/utils/delegation.ts

```diff
@@ -4,6 +4,8 @@ import { subgraphRequest } from '../utils';
 const SNAPSHOT_SUBGRAPH_URL = {
   '1': 'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot',
   '4': 'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-rinkeby',
+  '137': 'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-matic',
```

Comment:
This should be this url
```suggestion
  '137': 'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-polygon',
```